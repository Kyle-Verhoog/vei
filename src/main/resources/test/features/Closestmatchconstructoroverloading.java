public class Closestmatchconstructoroverloading {
  public Closestmatchconstructoroverloading() {}
  public Closestmatchconstructoroverloading(Object x, Object y) {}
  public Closestmatchconstructoroverloading(Object x, A y) {}
  public void m() {
    new A(new A(), new A());
  }
}