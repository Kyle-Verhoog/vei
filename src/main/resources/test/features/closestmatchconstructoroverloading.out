PUBLIC ┃ public
CLASS ┃ class
IDENTIFIER ┃ A
{ ┃ {
PUBLIC ┃ public
IDENTIFIER ┃ A
( ┃ (
) ┃ )
{ ┃ {
} ┃ }
PUBLIC ┃ public
IDENTIFIER ┃ A
( ┃ (
IDENTIFIER ┃ Object
IDENTIFIER ┃ x
, ┃ ,
IDENTIFIER ┃ Object
IDENTIFIER ┃ y
) ┃ )
{ ┃ {
} ┃ }
PUBLIC ┃ public
IDENTIFIER ┃ A
( ┃ (
IDENTIFIER ┃ Object
IDENTIFIER ┃ x
, ┃ ,
IDENTIFIER ┃ A
IDENTIFIER ┃ y
) ┃ )
{ ┃ {
} ┃ }
PUBLIC ┃ public
VOID ┃ void
IDENTIFIER ┃ m
( ┃ (
) ┃ )
{ ┃ {
NEW ┃ new
IDENTIFIER ┃ A
( ┃ (
NEW ┃ new
IDENTIFIER ┃ A
( ┃ (
) ┃ )
, ┃ ,
NEW ┃ new
IDENTIFIER ┃ A
( ┃ (
) ┃ )
) ┃ )
; ┃ ;
} ┃ }
} ┃ }